src/helm/benchmark/server.py

```diff
@@ -21,6 +21,7 @@
 
 @app.get("/config.js")
 def serve_config():
+    response.content_type = "application/javascript; charset=UTF-8"
```

Comment:
Interestingly, [RFC 9239](https://www.rfc-editor.org/rfc/rfc9239) says this should be `text/javascript` but it looks like this string is also acceptable and widely used.